### ISSUE TITLE
More straightforward and accessible way to add custom recipes

### ISSUE THREAD

**Issue by AntonBogun** [factoriolab/factoriolab#1182]
**Is your feature request related to a problem? Please describe.**
More straightforward and accessible way to add custom recipes. The use case is https://mods.factorio.com/mod/ZRecipeRandomizer, but also any other games where this would be useful.

**Describe the solution you'd like**
Possibly some way to neatly import an easily customizeable set of recipes. Also a small tutorial on what the data used by the project represents.

**Describe alternatives you've considered**
The alternative is to go dig through the code, which I'm willing to do myself but is obviously not for everyone.

**Additional context**
None.

**Comment by dcbroad3** [comment]
I assume you don't need to ever add items for your use case? The most difficult aspect of this would be adding icons, but recipes can inherit icons from existing items.

**Comment by AntonBogun** [comment]
Yeah, for my case specifically thankfully icons are already there, but in general it would be a good idea also to just add some placeholder icons for newly added items and then let to change them later.
